templates/AutoCompleteField.ss

```diff
@@ -1,2 +1,3 @@
 <input $AttributesHTML>
-<input type="hidden" name="$Name" value="$dataValue">
\ No newline at end of file
+<input type="hidden" name="$Name" value="$dataValue">
+<span class="value-holder<% if $Value %> has-value<% end_if %>">Selected: <em class="value" data-empty-val="(none)"><% if $Value %>$Value<% else %>(none)<% end_if %></em> <a href="#" class="clear">Clear ×</a></span>
```

Comment:
`$Value` is automatically escaped for XML appropriately in this context and I didn't change the $Name or $dataValue vars - do they need to be cast with `.ATT`?